src/mongoFacade/bearing_info.py

```diff
@@ -12,32 +13,37 @@ def get_bearings_by_org_name_and_status(org_name, status):
         bearings = Bearing.objects(organization__organization_name=org_name, alert_status=status)
     return bearings
 
-
 def get_bearing_by_id(id):
-    return Bearing.objects.get(id=id)
-
+    try:
+        return Bearing.objects.get(id=id)
+    except:
+        return None, 400, HEADER
```

Comment:
`except` range too broad. the best practice would be to only catch an exception that you can handle. In this case, it would be `except DoesNotExist:`  which is from `mongoengine`